hollow/src/main/java/com/netflix/hollow/api/client/StaleHollowReferenceDetector.java

```diff
@@ -114,7 +114,7 @@ private synchronized void housekeeping() {
 
     public void startMonitoring() {
         if (monitor == null) {
-            monitor = new Thread(new Monitor(this));
+            monitor = new Thread(new Monitor(this), "hollow-stale-reference-detector");
```

Comment:
Just happened to notice this as I was here. I'll be extracting this change out of this PR into it's own PR.

Reply:
#402